PolyDock/PolyDock/src/pd/ecs/sys/tabsHeader/TabsClosing.cpp

```diff
@@ -0,0 +1,100 @@
+#include <pd/pch/PCH.h>
+#include <pd/ecs/sys/tabsHeader/TabsClosing.hpp>
+
+#include <pd/ecs/cmp/root/Input.hpp>
+#include <pd/ecs/cmp/tabsHeader/TabsClosing.hpp>
+#include <pd/ecs/cmp/tabsHeader/TabsHeaderWidget.hpp>
+#include <pd/ecs/cmp/tabsHeader/TabsHeader.hpp>
+#include <pd/ecs/cmp/tabsHeader/TabsRemoving.hpp>
+
+using namespace ::pd::ecs::cmp;
+
+namespace pd::ecs::sys
+{
+
+// ---------------------------------------------------------------------------------------------------------
+void CloseButtonHoverSystem::update(entt::registry& registry, entt::entity root) const
+{
+	if (const auto* inputComponent = registry.try_get<root::Input>(root))
+	{
+		auto view = registry.view<tabsHeader::Widget, tabsHeader::Component>();
```

Comment:
let's decide on one syntax for this view iteration, either horizontally or vertically

Reply:
What do you mean?